src/pages/waiting-room-to-car/cat-home-test/components/vehicle-checks-modal/vehicle-checks-modal.cat-home-test.page.ts

```diff
@@ -62,6 +56,7 @@ export class VehicleChecksCatHomeTestModal {
 
   showMeQuestions: VehicleChecksQuestion[];
   tellMeQuestions: VehicleChecksQuestion[];
+  categoryCode: CategoryCode;
```

Comment:
In most other places this would be TestCategory, it would save you casting it in the test file as well